store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/FederatedOperation.java

```diff
@@ -102,15 +103,12 @@ public FederatedOperation<INPUT, OUTPUT> mergeFunction(final BiFunction mergeFun
         return this;
     }
 
+    @JsonGetter("skipFailedFederatedExecution")
```

Comment:
This is exactly the same behaviour as before. What the field is set as initially (`false`) is the 'default'. We tell jackson to only serialise if not the default by using `@JsonInclude(value = JsonInclude.Include.NON_DEFAULT)`. This gives us the same results as if we'd returned `null`, as `null` is also not serialised by jackson. But this way we don't need a separate method, only an annotation.

The reason for making this change is only because the old method caused a SpotBugs violation by returning `null` but with with return type of `Boolean`.

Reply:
👍